test/prompt_test.sh

```diff
@@ -33,13 +33,13 @@ run_bash_prompt
   fixture_dir 'git'
 
     # has an exit code of 0
-    sexy_bash_prompt_is_on_git || echo '`sexy_bash_prompt_is_on_git`; $? != 0 in git directory' 1>&2
+    sexy_bash_prompt_is_on_git || echo "Line $LINENO: "'`sexy_bash_prompt_is_on_git` was `false` in git directory' 1>&2
```

Comment:
"was `false`" falls back into confusion a bit. Prob better to leave it as the original: `$?` check